### DISCUSSION TITLE
Weblate login working with ip and port URL. Not working with dns name: CSRF verification failure.

### DISCUSSION THREAD

**Discussion by joanesinurrietasmartpm** [WeblateOrg/weblate#9454] [Q&A]
Hello,

I installed weblate in a server and when I access to it with the ip and port, it works ok and I can login.

When I open the website with the DNS name, the web opens but when I try to login I got this error:

![image](https://github.com/WeblateOrg/weblate/assets/131172312/2a91ad25-5c06-42f2-af54-8c66a2c559f8)

Permiso denegado
Falló la verificación de CSRF. Se interrumpió la petición.

I don't know if I have to add the custom name to some parameter.

Any help?

Thanks in advance.

**Comment by nijel** [comment]
See https://docs.djangoproject.com/en/4.2/ref/csrf/ for info how this works.
